keyboards/a_dux/README.md

```diff
@@ -0,0 +1,30 @@
+# Architeuthis Dux
+
+![Architeuthis Dux](https://raw.githubusercontent.com/tapioki/cephalopoda/main/Images/architeuthis_dux.png)
+
+AKA A. dux, A.D., "Giant squid"
+
+34 keys, same stagger as [D. gigas](https://github.com/tapioki/cephalopoda/tree/main/Dosidicus%20gigas), but with two thumb keys included. Pinky, ring, index, and inner columns splayed at 15, 5, 0, -5, -5 degrees.  SplitKB Tenting Puck support.  All PCB files are available on the [project's github page](https://github.com/tapioki/cephalopoda/tree/main/Architeuthis%20dux)
+
+Thank you [Perce](https://madebyperce.com/) for the logo.
+
+### Cases can be found here:
+
+* [Flat Case](https://github.com/madebyperce/aduxcase)
+* [Tented Case](https://github.com/jdart/adux-tent)
+* [Case with Plates](https://github.com/sadekbaroudi/cephalopoda/tree/main/Architeuthis%20dux/case)
+
+### Keyboard Info
+
+* Keyboard Maintainer: [tapioki](https://github.com/tapioki)
+* Hardware Supported: Architeuthis Dux
+* Hardware Availability: [GitHub](https://github.com/tapioki/cephalopoda/tree/main/Architeuthis%20dux)
+
+Make examples for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
This keyboard doesn't have a physical reset button, so I described how to reset the microcontroller by shorting the reset and round pins.  Let me know if that works!